src/ast/mod.rs

```diff
@@ -645,19 +646,24 @@ impl fmt::Display for Statement {
                 external,
                 file_format,
                 location,
+                query,
             } => {
+                let include_parens = query.is_none() || !columns.is_empty();
```

Comment:
AFAIK `CREATE TABLE t () AS ...` doesn't parse (tested in postgresql) but `create table t (a int) as 1 works`. `create table t ()` parses though, so that we'll want to allow. 

We can add a test for this to make sure it doensn't create the `CREATE TABLE t () AS ...`

Reply:
You didn't explain the reason for `query.is_none()`. I think the parens should be printed IFF `!columns.is_empty()`.

Reply:
I think you still want to allow the statement `create table t ()`. I believe `create table t` will not be valid (at least in postgresql). Only printing the parens of non empty columns will not cover that case.

Reply:
Ah, It's so surprising that `create table t()` is valid in postgres, that I misread your comment the first time, sorry! I remember now that we do support that (#94). Could you add a code comment explaining the need to print the empty parens if `query.is_none()`?

Reply:
Thanks for adding the comment! Re-reading it, I thought that maybe printing the parens in a separate block following the `write!("CREATE {external}TABLE {if_not_exists}{name} ", ...)` would be clearer:

    if !columns.is_empty() || !constraints.is_empty() {
        write!(f, "({}", display_comma_separated(columns))?;
        if !columns.is_empty() && !constraints.is_empty() {
            write!(f, ", ")?;
        }
        write!(f, "{})", display_comma_separated(constraints))?;
    } else if query.is_none() {
         // PostgreSQL allows `CREATE TABLE t();`, but requires empty parens
        write!(f, "()")?;
    }